diff --git a/server.js b/server.js
index 4baf7ca..70cf769 100644
--- a/server.js
+++ b/server.js
@@ -6,13 +6,12 @@ const SequelizeStore = require('connect-session-sequelize')(session.Store);

 const routes = require('./controllers/');
 const sequelize = require('./config/connection');
-// const helpers = require('./utils/helpers');
+const helpers = require('./utils/helpers');

 const app = express();
 const PORT = process.env.PORT || 3001;

-const hbs = exphbs.create();
-
+const hbs = exphbs.create({helpers});

 app.engine('handlebars', hbs.engine);
 app.set('view engine', 'handlebars');
diff --git a/utils/helpers.js b/utils/helpers.js
index 7ea053b..f955062 100644
--- a/utils/helpers.js
+++ b/utils/helpers.js
@@ -1,4 +1,6 @@
 module.exports = {
-
-  };
+  print_object: (object) => {
+    return JSON.stringify(object);
+  }
+};

\ No newline at end of file
diff --git a/views/battle.handlebars b/views/battle.handlebars
index 143f7b2..ee33cdd 100644
--- a/views/battle.handlebars
+++ b/views/battle.handlebars
@@ -38,7 +38,9 @@
 <div class="reset">
 <button class="reset-button">Try Again?</button>
 </div>
-
+<script>
+    window.classes = {{{print_object classes}}};
+</script>
 <script src="/js/startScreen.js"></script>
 <script src="/js/checks.js" type="module"></script>
 <script src="/js/buff.js" type="module"></script>